tutorials/physics/physics_introduction.rst

```diff
@@ -180,6 +180,15 @@ would be as follows::
     # (2^(1-1)) + (2^(3-1)) + (2^(4-1)) = 1 + 4 + 8 = 13
     pow(2, 1-1) + pow(2, 3-1) + pow(2, 4-1)
 
+You can also set bits independently by calling ``set_collision_layer_value(layer_number, value)``
+or ``set_collision_mask_value(layer_number, value)`` on any given :ref:`CollisionObject <class_CollisionObject2D>` as follows::
+
+    # Example: Setting mask value to enable layers 1, 3, and 4.
+
+    var collider: CollisionObject2D = $CollisionObject2D  # Any given collider.
+    collider.set_mask_layer_value(1, true)
+    collider.set_mask_layer_value(3, true)
+    collider.set_mask_layer_value(4, true)
```

Comment:
```suggestion
    collider.set_collision_mask_value(1, true)
    collider.set_collision_mask_value(3, true)
    collider.set_collision_mask_value(4, true)
```
The function name was wrong in the example, there is no `set_mask_layer_value()` function